Automattic/wp-calypso
# Media library: add a token failure error state

As part of an upcoming Picasa => Google Photos migration, introduce an additional error state to the media library.

The `keyring_token_error` will indicate that an authentication error occurred when using an external service. We detect this when a media operation has been performed, and convert it to `SERVICE_AUTH_FAILED`. This can then be used elsewhere for additional migration logic.

Note there are a bunch of linting changes in this PR because it's forcing me to make them

#### Testing instructions

The change is fairly small and adds an extra case to a switch. The existing unit tests + new test should cover the change.

No visible changes should be seen.
